simulation/src/main.cpp

```diff
@@ -7,10 +7,10 @@
 int main(int argc, char **argv)
 {
     // Check CMD arguments
-    if(argc != 3)
+    if(argc != 4)
```

Comment:
added new command line argument to set the number of rounds you want to run. This allows for you to run like 100 rounds of the game and count how many time each player won.